### PR TITLE
webgpu: Use uniforms for PadV2 paddings

### PR DESCRIPTION
**Pull request by qjia7**
This PR uses uniforms for PadV2 paddings. Meanwhile, it does a small
optimization to directly return the identity of the input if all
paddings are zero.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4966)

### CODE REVIEW

**Comment by qjia7** [comment]
@axinging  @xhcao @haoyunfeix @gyagp, PTAL

File: tfjs-backend-webgpu/src/kernels/pad_webgpu.ts [hunk 1]
```diff
@@ -42,19 +42,21 @@ export class PadProgram implements WebGPUProgram {
     this.dispatch = computeDispatch(
         this.dispatchLayout, this.outputShape, this.workGroupSize,
         [this.workPerThread, 1, 1]);
-
+    paddings.map((_, i) => this.uniforms += ` ivec2 pad${i};`);
     this.xShape = xShape;
     this.paddings = paddings;
```
Comment (axinging) [inline tfjs-backend-webgpu/src/kernels/pad_webgpu.ts:47]: LGTM.
nit: this.paddings = paddings.map((_, i) => this.uniforms += ` ivec2 pad${i};`);?

Comment (qjia7) [reply tfjs-backend-webgpu/src/kernels/pad_webgpu.ts:47]: I don't need to change the value of `this.paddings`. I suppose above assignment will change the content of `this.paddings`.

**Comment by qjia7** [comment]
@lina128  @jinjingforever @kainino0x Please take a look, thanks.

**Review by jinjingforever — APPROVED** [review]

**Pull request merged**